src/chapters/classes/exercises.rst

```diff
@@ -0,0 +1,27 @@
+.. _classes-exercises-part1:
+
+Exercises: Classes and Objects
+==============================
+
+Before you get started, make sure you have forked and cloned the `starter code <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn3and4classes>`_.
+
+#. Open up the file, ``Student.cs``, and add all of the necessary getters and
+   setters. Think about the access level each field and method should have, and
+   try reducing the access level of at least one setter to less than public.
+#. Instantiate the ``Student`` class using yourself as the student. For the
```

Comment:
we might also want to add "In Program.cs" to the beginning of this item to be clear what it is for

Reply:
Done!